src/test/java/io/cdap/plugin/table/streaming/TableStreamingSourceConfigTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.table.streaming;
+
+import io.cdap.cdap.etl.api.validation.CauseAttributes;
+import io.cdap.cdap.etl.api.validation.ValidationFailure;
+import io.cdap.cdap.etl.mock.validation.MockFailureCollector;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class TableStreamingSourceConfigTest {
+  private static final String MOCK_STAGE = "mockStage";
+  private static final TableStreamingSourceConfig VALID_CONFIG = new TableStreamingSourceConfig(
+    "test",
+    "{\"type\":\"record\",\"name\":\"etlSchemaBody\",\"fields\":[{\"name\":\"t\"," +
```

Comment:
Please create a schema object as do `.toString()` for it. It would be easier to maintain.

Reply:
Done.